go-controller/pkg/ovn/controller/services/services_controller_test.go

```diff
@@ -684,6 +684,7 @@ func nodeConfig(nodeName string, nodeIP string) *nodeInfo {
 		gatewayRouterName: nodeGWRouterName(nodeName),
 		switchName:        nodeSwitchName(nodeName),
 		chassisID:         nodeName,
+		zone:              types.OvnDefaultZone,
```

Comment:
can we add tests where nodes are on different zones and if the filtering of endpoints etc are correctly done and not processed in the cases where nodes are not in local zones?

Reply:
I'll try but after updating all the review comments from this round

Reply:
@dceara @jcaamano   what happens presently with the chassis template var when a node is deleted.
Does the service controller delete the chassis template var entry for the deleted node ?

In the test case added here - https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/controller/services/services_controller_test.go#L447  why is that the 'dbStateAfterDeleting' expects the nodeIPTemplate for the deleted node ?

When a  zone moves from a  zone - az1 to a remote zone az2,  I'd expect the az1 ncm service controller to delete the chassis template var row for this node.  I tested it and it is working as expected in my kind setup.

But in the test case (which I tried to add), the entry is still there.

And also the link I shared above,  I was expecting the nodeIPTemplate for the deleted node to be removed.